broadinstitute/pearl
# AR-71 adding editing for consent and prereg forms

This extends the editing capabilities for surveys to consent forms and prereg forms.

TO TEST:
1. nuke/reset your db
2. run `scripts/populate_setup.sh` and `scripts/populate_portal.sh ourhealth`
3. go to `http://localhost:3000/ourhealth/studies/ourheart/env/sandbox`
4. confirm prereg and consent appear.
5. edit and save both the prereg and consent form.  
6. Confirm the updated versions are saved and attached to the study environment
